cmd/crc/cmd/status.go

```diff
@@ -14,7 +15,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const jsonFormat = "json"
+
+var (
+	outputFormat string
+)
+
 func init() {
+	statusCmd.Flags().StringVarP(&outputFormat, "output", "o", "", "Output format. One of: json|wide")
```

Comment:
The `wide` format can be removed, as its not been implemented.

Reply:
Yes good point. What could be the name of the default format ? For kubectl, it's wide if I'm correct.

Reply:
`kubectl` wide gives more information, then without a `-o` flag which is the default. i don't think we need to provide the default also as option for the output flag.

Reply:
I removed wide and just let json. Is it better? `Output format. One of: json` ?